review/ha_basic.toml

```diff
@@ -0,0 +1,44 @@
+[server]
+hostname = "wso2is.example.com"
+proxy_port = "443"
+local_ip = "192.168.1.1"
+
+[super_admin]
+username = "admin"
+password = "admin"
+
+[user_store]
+type = "read_write_ldap"
+connection_url = "ldap://ldap.example.com:389"
+connection_name = "uid=admin,ou=wso2is"
+connection_password = "$secret{ldap_password}"
+base_dn = "dc=example,dc=com"
+
+[database.core_db]
+type = "mysql"
+hostname = "mysql.example.com"
+database = "wso2regdb"
+password = "$secret{db_password}"
+
+[database.identity_db]
+type = "mysql"
+hostname = "mysql.example.com"
+database = "wso2isdb"
+password = "$secret{db_password}"
+
+[keystore.tls]
+name = "keystore.jks"
+password = "$secret{keystore_password}"
+
+[keystore.crypto]
+name = "keystore.jks"
+password = "$secret{key_password}"
+
+[cluster]
+scheme = "wka"
+members = ["192.168.1.1:4000","192.168.1.2:4000"]
```

Comment:
port 4000 is not configured for local. so need to derive the port 4000 on remote also.

Reply:
Some customers seems to use different port for this other than 4000. That's why we thought of including the port also.